src/providers/wfs/qgswfsdatasourceuri.h

```diff
@@ -135,6 +135,17 @@ class QgsWFSDataSourceURI
     //! Returns authorization parameters
     const QgsAuthorizationSettings &auth() const { return mAuth; }
 
+    //! How to analyze DescribeFeatureType response
+    enum class FeatureMode
+    {
+      DEFAULT,          //! If the server supports transaction, same as SIMPLE_FEATURE. Otherwise COMPLEX_FEATURE
+      SIMPLE_FEATURES,  //! Analyze DescribeFeatureType response with QGIS built-in Simple Feature analyzer
+      COMPLEX_FEATURES, //! Analyze DescribeFeatureType response with OGR GMLAS Complex Feature analyzer
```

Comment:
```suggestion
      Default,          //! If the server supports transaction, same as SIMPLE_FEATURE. Otherwise COMPLEX_FEATURE
      SimpleFeatures,  //! Analyze DescribeFeatureType response with QGIS built-in Simple Feature analyzer
      ComplexFeatures, //! Analyze DescribeFeatureType response with OGR GMLAS Complex Feature analyzer
```

Reply:
fixed